src/main/java/wis/util/Printer.java

```diff
@@ -0,0 +1,92 @@
+package wis.util;
+
+import wis.Action;
+import wis.task.Task;
+import wis.task.TaskList;
+
+public class Printer {
+    public static void printDecorator() {
+        System.out.println("    ____________________________________________________________");
+    }
+
+    public static void println(String string) {
+        System.out.println("     " + string);
+    }
+
+    public static void printlnFurtherIndent(String string) {
+        System.out.println("       " + string);
+    }
+
+    public static void printActionAttach(Action action, Task task, TaskList tasks) {
+        switch (action) {
+            case ADD_TODO:
+            case ADD_DEADLINE:
+            case ADD_EVENT:
+                printDecorator();
+                Printer.println("Got it. I've added this task:");
+                Printer.printlnFurtherIndent(task.toString());
+                tasks.printTaskCount();
+                printDecorator();
+                break;
+            case DELETE:
+                printDecorator();
+                Printer.println("Noted. I've removed this task:");
+                Printer.printlnFurtherIndent(task.toString());
+                tasks.printTaskCount();
+                printDecorator();
+                break;
+            default:
+                throw new IllegalArgumentException("Illegal action argument provided.\n");
```

Comment:
`case` statements should be inline with `switch` statement according to the coding guideline

Reference: https://se-education.org/guides/conventions/java/intermediate.html#layout
<img width="989" alt="image" src="https://github.com/nus-cs2103-AY2324S2/ip/assets/44436551/6fd32b30-2e85-4788-8240-e56eaca33058">

Reply:
Noted. Thank you! (I forgot to change this)